src/common_types/constants.hpp

```diff
@@ -295,10 +295,16 @@ constexpr inline uint8_t standard_pause = log | popup | pause;
 
 enum class game_mode {
 	pick_nation,
-	single_player,
+	in_game,
```

Comment:
The reason to keep single player here is that the game loop won't work the same way. In single player, the game loop advances the days automatically based on time. In multiplayer it must do so only when it receives a command to advance the day. So these are not differences only in networking